File: salg01.py
def sep(s):
    a=[]
    b=''
    for i in list(s):
        if(i==' ' or i=='\n'):
            a.append(int(b))
            b=''
        b=b+i
    return(a)

t=int(input())
for k in range(t):
    L=int(input())
    x=input()
    n=sep(x)
    maxm=n[0]
    for i in range(1,L):
        if(n[i]>maxm):
            p=0
            ans=i+1
    if(p==0):
        print(ans)
    else:
        print(1)

File: salg01.c
#include<stdio.h>
main()
{
    int test_cases,i,n,L,k,maxm,p,ans;
    scanf("%d",&test_cases);
    for(k=0;k<test_cases;k++)
    {
        scanf("%d",&L);
        scanf("%d",&maxm);
        p=1;
        for(i=1;i<L;i++)
        {
            scanf("%d",&n);
            if(n>maxm)
            {
                p=0;
                ans=i+1;
                maxm=n;
            }
        }

        if(p==0) printf("%d\n",ans);
        else printf("1\n");
    }

return 0;
}
